sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -141,7 +141,7 @@ statement
         (LIKE? pattern=STRING)?                                        #showTables
     | SHOW TABLE EXTENDED ((FROM | IN) db=identifier)?
         LIKE pattern=STRING partitionSpec?                             #showTable
-    | SHOW DATABASES (LIKE pattern=STRING)?                            #showDatabases
+    | SHOW DATABASES (LIKE? pattern=STRING)?                            #showDatabases
```

Comment:
We are following Hive. Any change was made in the recent releases of Hive?

Reply:
Hive did not change, but I think spark can change the next.

Reply:
@guoxiaolongzte . MySQL also doesn't work like that. Do you have any reference for that syntax?

Reply:
No, I just saw like show tables like can be removed. so I think show databases like can also be removed. Just think it is removed, the operation is more convenient.

Reply:
Impala is the only reference I can find https://www.cloudera.com/documentation/enterprise/5-10-x/topics/impala_show.html

To be consistent with the other SHOW function, we can make LIKE optional?

Reply:
@gatorsmile  @dongjoon-hyun 
I think we can make LIKE optional.